generators/client/templates/react/src/main/webapp/app/shared/layout/password/password-strength-bar.css.ejs

```diff
@@ -28,7 +28,7 @@ ul#strengthBar {
   vertical-align: 2px;
 }
 
-.point:last {
+.point:last-child {
```

Comment:
missing for app.scss.ejs, password-strength-bar.css.ejs and password-strength-bar.scss.ejs

Reply:
in react client

Reply:
actually the app.scss and app.css has duplicated style def from password-strength-bar, can you please remove them as well

Reply:
Removed from app.scss. I'm not seeing the same duplication in app.css.

Reply:
Also updated the value in password-strengh-bar.scss.ejs for the react client.

Reply:
And did a global search for ":last" in .ejs files and they've all got -child now